build-tools/heroku/generate-lezer-parsers

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eu
+
+# Generate the lezer parsers from the grammars specified in  highlighting/lezer-grammars/
+for level in $(seq 1 18);
```

Comment:
Can't we do this via pattern matching? That way, when we add levels, we don't need to remember to update this number here.

Maybe something like:

```bash
for grammarFile in ../../highlighting/lezer-grammers/level*.grammar; do
  baseName=$(basename "$grammarFile")
  tsFile=${baseName%.grammar}-parser.ts
  npx lezer-generator --typeScript "$grammarFile" -o ../../static/js/lezer-parser/$tsFile
done
```

Reply:
Great idea!!! I'm very new to bash scripting, so this didn't occur to me.